rsync_to_esmeval.sh

```diff
@@ -0,0 +1,3 @@
+rsync -av bgcval2/CompareReports2/* /gws/nopw/j04/esmeval/public/CompareReports/bgcval2/.
+rsync -av CompareReports2/* /gws/nopw/j04/esmeval/public/CompareReports/bgcval2/.
+
```

Comment:
add a memtion that one needs access to `esmeval` maybe

Reply:
Maybe we should add a $USER path there?

Reply:
yeah - but that means that user needs to have a user dir there